application-admintools-default/src/main/resources/templates/wikiSizeTemplate.vm

```diff
@@ -57,9 +57,12 @@
           <li><a href="#pagesOverNumberOfComments" data-toggle="modal" data-target="#pagesOverNumberOfComments">
             $escapetool.xml($services.localization.render('adminTools.dashboard.instanceUsage.specific.comments',
             [$spamLimit]))</a></li>
+          <li><a href="#viewWikisRecycleBinModal" data-toggle="modal" data-target="#viewWikisRecycleBinModal">
```

Comment:
You are also missing a title for this link

Reply:
You should add a title to this element, so when the user hovers over it he will get more info about what it does